### ISSUE TITLE
dicom.Write: The Pixel data only supports OW type

### ISSUE THREAD

**Issue by ChaoStraxcorp** [suyashkumar/dicom#299]
When I try to write a Dicom dataset to a file, I encountered the following error:
```
ERROR dicomio.veryifyElement: VR mismatch for tag (7fe0,0010)[PixelData]. Element.VR=OB, but DICOM standard defines VR to be OW
```
However, the pixel data should support both OW and OB types:
https://dicom.nema.org/medical/dicom/current/output/chtml/part05/chapter_8.html

In [pkg/tag/tag_definitions.go](https://github.com/suyashkumar/dicom/blob/main/pkg/tag/tag_definitions.go), the PixelData can only be OW.

**Comment by suyashkumar** [comment]
Thanks for logging this. I think as part of the updates we'd like to make to the tag definitions (e.g. a follow on to #169) we will allow support for tags that can have multiple VRs. Definitely an important feature, and we'll take a look at this in the next few months!

**Comment by suyashkumar** [comment]
Closed by #316
